site/docs/hosting/comparison.md

```diff
@@ -29,20 +29,21 @@ We therefore recommend you to use a different kind of hosting for more complex b
-| Scaleway Containers    | Free       | €0.10/100K GB-s, €1.0/100K vCPU-s                                     | [400K GB-s, 200K vCPU-s/mo](https://www.scaleway.com/en/pricing/#serverless-containers) | ❓       | ❓                           | ❓   |                                |
-| Vercel Edge Functions  | Free       | $20/mo subscription for 500K                                          | [100K req/day](https://vercel.com/pricing)                                              | ❓       | ❓                           | ❓   |                                |
-| serverless.com         | Free       |                                                                       |                                                                                         | ❓       | ❓                           | ❓   |                                |
-| DigitalOcean Apps      | $5         |                                                                       |                                                                                         | ❓       | ❓                           | ❓   | Not tested                     |
-| Fastly Compute@Edge    |            |                                                                       |                                                                                         | ❓       | ❓                           | ❓   |                                |
+| Name                   | Min. price | Pricing                                                                  | Limits                                                                                  | Node.js | Deno                        | Web | Notes                          |
+| ---------------------- | ---------- | ------------------------------------------------------------------------ | --------------------------------------------------------------------------------------- | ------- | --------------------------- | --- | ------------------------------ |
+| Deta                   | Free       | No paid plans yet                                                        | No specific limits                                                                      | ✅       | ❓                           | ❓   |                                |
+| Deno Deploy            | Free       | $10/mo subscription for 5M req and 100 GB; $2/1M req, $0.3/GB network    | [100K req/day, 100 GB/mo, 10 ms CPU-time limit](https://deno.com/deploy/pricing)        | ❌       | ✅                           | ❌   |                                |
+| Fly                    | Free       | $1.94/mo subscription for shared-cpu-1x and 256 MB RAM, $0.02/GB network | [3 shared-cpu-1x 256mb VMs, 160GB/mo, 3GB storage](https://fly.io/docs/about/pricing/)  | ✅       | ✅                           | ❓   |                                |
```

Comment:
Is that shared-cpu-1x a Fly term or a general one meaning a single shared CPU?

Reply:
It seems to be a fly term: https://fly.io/blog/new-vms-more-ram-extra-cpu-and-a-dollar-menu/

Reply:
Yup. That's a fly term: https://fly.io/docs/about/pricing/